src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -310,9 +310,22 @@ export class RunParametersAction extends TooltipFromLabelAction {
 		let defaultParameters = new Map<string, string>();
 		editor.cells.forEach(cell => {
 			if (cell.isParameter) {
+				if (!cell.source) {
```

Comment:
What if the source is all whitespace or invalid parameters?

Reply:
Should be covered by checking the regex test for each line in the parameter cell. Setting that emptyParameterCell to then only send the info message and not execute the notebook as well.